db/migrate/20150419175648_menu_items_orders.rb

```diff
@@ -0,0 +1,8 @@
+class MenuItemsOrders < ActiveRecord::Migration
+  def change
+    create_table :menu_items_orders do |t|
+      t.belongs_to :order, index: true
+      t.belongs_to :menu_item, index: true
+    end
+  end
+end
```

Comment:
For the record, we should least test the join table name. ;-). Using shoulda_matchers, we can use:
``` 
it { is_expected.to have_and_belong_to_many(:orders), join_table_name: :menu_items_orders}
´´´